certs/.gitignore

```diff
@@ -0,0 +1,2 @@
+*
+!.gitignore
```

Comment:
erm? :D

also, what is that directory for? Could you add appropriate README? Otherwise I'm unsuure if it should be added to .e.g `MANIFEST.in`